packages/helpers/classes/attachment.js

```diff
@@ -40,38 +40,65 @@ class Attachment {
     data = toCamelCase(data);
 
     //Extract properties from data
-    const {content, filename, type, disposition, contentId, file} = data;
-
-    let fileContent = !file ? content : this.readFile(file);
+    const {
+      content,
+      filename,
+      type,
+      disposition,
+      contentId,
+      filePath
+    } = data;
+
+    if ((typeof content !== 'undefined') && (typeof filePath !== 'undefined')) {
+      throw new Error(
+        'The props \'content\' and \'filePath\' cannot be used together.'
+      );
+    }
 
     //Set data
-    this.setContent(fileContent);
     this.setFilename(filename);
     this.setType(type);
     this.setDisposition(disposition);
     this.setContentId(contentId);
+
+    if (filePath) {
+      const fileContent = this.readFile(filePath);
+      this.setContent(fileContent);
+      return;
+    }
+
+    this.setContent(content);
   }
 
   /**
    * Read a file and return its content as base64
    */
-  readFile(file) {
-    let encoding = !file.encoding ? 'utf8' : file.encoding;
-    let data = fs.readFileSync(file.path);
-    let buff = Buffer.from(data, encoding);
-    return buff.toString('base64');
+  readFile(path) {
+    return fs.readFileSync(path);
   }
 
   /**
    * Set content
    */
   setContent(content) {
-    if (typeof content === 'undefined') {
+    const contentType = typeof content;
+
+    if (contentType === 'undefined') {
       return;
-    }
-    if (typeof content !== 'string') {
+    } else if (contentType === 'object'
```

Comment:
Could you please install ESLint for your code editor and run it on your code to ensure the code style is the same to the rest of the code base?

Reply:
Sure, I was using the StandardJS

Reply:
Have you used ESLint on your changes? The code style is still different.